### PR TITLE
Open PromCon 2022 EU

### PR DESCRIPTION
**Pull request by RichiH**
Signed-off-by: Richard Hartmann <richih@richih.org>

### CODE REVIEW

File: content/2022-munich/giggity.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": 2019102900,
+    "url": "https://promcon.io/2019-munich/schedule.xml",
+    "title": "PromCon EU 2019",
+        "start": "2019-11-07",
+        "end": "2018-11-08",
```
Comment (juliusv) [inline content/2022-munich/giggity.json:6]: Looks like all these days here need updating, also in the website link below?

File: content/2022-munich/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+title: Overview
+---
+
+## Overview
+
+PromCon EU 2019 is the seventh conference fully dedicated to the
```
Comment (juliusv) [inline content/2022-munich/index.md:7]: Still everything 2019 here...

File: content/2022-munich/submit.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: Submit Talk
+---
+
+## Submit a talk
+
+The PromCon Call for Papers (CfP) is open for talk submissions until 23:59 UTC on
+Sunday, July 31st, 2019, and we encourage you to submit a talk!
```
Comment (juliusv) [inline content/2022-munich/submit.md:8]: Still 2019

File: content/2022-munich/submit.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+<a class="btn btn-default btn-lg" href="" target="_blank" role="button">
+  <i class="fa fa-commenting"></i>&nbsp;&nbsp;Submit Talk
```
Comment (juliusv) [inline content/2022-munich/submit.md:11]: The button currently doesn't do anything?

File: lib/helpers/conferences.rb [hunk 5]
```diff
@@ -118,6 +118,22 @@ def conferences
       },
       Banner.new('losangeles-banner', 'Cameron Venti', 'https://unsplash.com/photos/0YWaDPylkYA'),
     ),
+    Conference.new(
+      'PromCon EU 2022',
+      'The Prometheus conference — November 08 - 09 in Munich',
+      '/2019-munich/',
```
Comment (juliusv) [inline lib/helpers/conferences.rb:124]: 2019

File: content/2022-munich/giggity.json (hunk 1 above)
Comment (RichiH) [reply content/2022-munich/giggity.json:6]: Thanks!

File: content/2022-munich/submit.md (hunk 3 above)
Comment (RichiH) [reply content/2022-munich/submit.md:8]: Thanks!

File: lib/helpers/conferences.rb (hunk 5 above)
Comment (RichiH) [reply lib/helpers/conferences.rb:124]: Thanks!

File: content/2022-munich/submit.md (hunk 4 above)
Comment (RichiH) [reply content/2022-munich/submit.md]: Thanks!

**Comment by RichiH** [comment]
Hmm, "fun" fact: We had broken dates in giggity for 2019 as well.

File: content/2022-munich/index.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+professional connections around systems and service monitoring.
+
+Get an impression of PromCon EU 2019:
+
+<%= youtube_player("TUbj9ykgafM") %>
+
+## Venue Information
+
+[Alte Kongresshalle](https://www.altekongresshalle.de/)
+Am Bavariapark 14
```
Comment (juliusv) [inline content/2022-munich/index.md:23]: In HTML, this doesn't show up with line breaks. You'll need to add `<br />` after each line.

**Comment by juliusv** [comment]
:+1: besides the address line-break nit

**Pull request merged**